SEED-platform/seed
# 2384 espm import single property template

#### Any background context you want to provide?
When an espm template has only one property, it is returned as a dictionary object as SEED expected a list of records. Import espm with single property failed to proceed to data mapping.

#### What's this PR do?
Append single property espm template/record to a list object for upload.

#### How should this be manually tested?
Create an espm with 1 property. Import a espm template with one property and expect it to be successfully imported for mapping.

#### What are the relevant tickets?
#2384 

#### Screenshots (if appropriate)
1. Create an espm with 1 property:
<img width="997" alt="Screen Shot 2020-08-29 at 3 11 37 PM" src="https://user-images.githubusercontent.com/49968203/91646206-04d71b00-ea0a-11ea-8d04-bf4ef449f1d5.png">

2. From SEED, import espm:
<img width="626" alt="Screen Shot 2020-08-29 at 3 08 48 PM" src="https://user-images.githubusercontent.com/49968203/91646174-9eea9380-ea09-11ea-8012-0d62cda72abc.png">

3. Expect to see success:
<img width="622" alt="Screen Shot 2020-08-29 at 3 10 54 PM" src="https://user-images.githubusercontent.com/49968203/91646208-0b659280-ea0a-11ea-8a27-c83272e6f550.png">
